src/RadixDlt.CoreApiSdk/Model/Interfaces.cs

```diff
@@ -82,6 +82,11 @@ public interface IEntityAddressPointer
     public IEnumerable<string> GetEntityAddresses();
 }
 
+public interface INonFungibleGlobalIdPointer
+{
+    public NonFungibleGlobalId GetNonFungibleId();
```

Comment:
Super minor but if we distinguish between local(?) vs global NFIDs maybe we should name the method `GetNonFungibleGlobalId()` as well?